<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regis</class>
 <widget class="QWidget" name="regis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="mailboxline">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>143</y>
     <width>142</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="rPasswordline">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>142</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>37</y>
     <width>45</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>37</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>检查用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>209</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>241</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>同意注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="tellabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>176</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>手机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telline">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>176</y>
     <width>142</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="womanradioButton">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>209</y>
     <width>93</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>241</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>直接登陆</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordline">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>77</y>
     <width>142</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameline">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>142</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="manradioButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>209</y>
     <width>42</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordlabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>77</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="mailboxlabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>143</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="rPasswordlabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>重复密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
